core/pom.xml

```diff
@@ -78,6 +91,7 @@
                           io.stargate.core.*,
                           com.codahale.metrics,
                           com.codahale.metrics.*,
+                          io.micrometer.*
```

Comment:
Do we need to export the whole of `io.micrometer.*`? I suppose, ideally, we'd export only the packages required for registering meters... Would it be possible to restrict exports to the bare minimum?

`com.codahale.metrics` is there historically, but for new changes it would be nice to avoid overly wide exports, I think.

If it turns out to be too messy, current form is ok, though.

Reply:
So for the osgi config this does not work.. I need to export this and have in other modules the import as well as the  are provided dependency in the health checker.. Otherwise it's either linkage error as classes are different or class not found.. If you think this is something wrong, we can check again. But it has to be the same reason the `com.codahale.metrics` are also exported here and why other modules are importing them..

Did I said already I hate OSGi? :sob: